Block/Account/CompanyData.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Devall\Company\Block\Account;
+
+use Magento\Framework\View\Element\Template;
+use Magento\Customer\Api\CustomerRepositoryInterface;
+use Magento\Customer\Model\Session as CustomerSession;
+use Devall\Company\Model\CompanyRepository;
+use Devall\Company\Model\ResourceModel\Company\CollectionFactory;
+use Devall\Company\Model\Company as CompanyModel;
+
+class CompanyData extends Template {
+
+    const REST_API_URL = '/rest/V1/devall_company';
+
+    /**
+     * @var Devall\Company\Model\ResourceModel\Company\Collection
```

Comment:
Undefined Class error. Also, just import this class. You've imported all the others.. looks chaotic.